crates/proposer-service/src/lib.rs

```diff
@@ -29,24 +30,23 @@ pub mod error;
 #[cfg(test)]
 mod tests;
 
-const AGGREGATION_VKEY_HASH: VKeyHash =
-    VKeyHash::from_hash_u32(aggchain_proof_core::AGGREGATION_VKEY_HASH);
+pub const AGGREGATION_ELF: &[u8] = include_bytes!("../elf/aggregation-elf");
 
 pub struct ProposerService<L1Rpc, ProposerClient> {
     pub client: Arc<ProposerClient>,
 
     pub l1_rpc: Arc<L1Rpc>,
 
-    /// Expected aggregated span proof verification key.
-    aggregation_vkey_hash: VKeyHash,
+    /// Aggregated span proof verification key.
+    aggregation_vkey_hash: SP1VerifyingKey,
```

Comment:
```suggestion
    aggregation_vkey: SP1VerifyingKey,
```

(plus ensuring that what we affect to it elswhere is named accordingly as well)